src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundKeywordsAndOperators.java

```diff
@@ -156,10 +156,8 @@ private void divTest()
     /** assert statement test */
     public void assertTest()
     {
-        // OK
         assert true;
 
-        // OK
```

Comment:
Lines deletion

Reply:
next time, to avoid updated in Test class, you can keep empty line.
But as you did removal already , it is also good.